storage/storagedriver/select-storage-driver.md

```diff
@@ -26,19 +26,28 @@ this decision, there are three high-level factors to consider:
   explicitly configured, assuming that the prerequisites for that storage driver
   are met:
 
-  - If possible, the storage driver with the least amount of configuration is
-    used, such as `btrfs` or `zfs`. Each of these relies on the backing
-    filesystem being configured correctly.
-
-  - Otherwise, try to use the storage driver with the best overall performance
-    and stability in the most usual scenarios.
-
-    - `overlay2` is preferred, followed by `overlay`. Neither of these requires
-      extra configuration. `overlay2` is the default choice for Docker CE.
+  - Try to use the storage driver with the best overall performance and stability
+    in the most usual scenarios.
 
+    - `overlay2` is the preferred storage driver, for all currently supported
+      Linux distributions, and requires no extra configuration.
+    - `aufs` is the preferred storage driver for Docker 18.06 and older, when
+      running on Ubuntu 14.04 on kernel 3.13 (which has no support for `overlay2`.
     - `devicemapper` is next, but requires `direct-lvm` for production
       environments, because `loopback-lvm`, while zero-configuration, has very
-      poor performance.
+      poor performance. `devicemapper` was the recommended storage driver for
+      CentOS and RHEL, as their kernel version did not support `overlay2`. However,
+      current versions of CentOS and RHEL now have support for `overlay2`, and
+      is now the recommended driver.
+    - The `btrfs` and `zfs` storage drivers are used if they are the backing
+      filesystem (the filesystem of the host on which Docker is installed).
+      These filesystems allow for advanced options, such as creating "snapshots",
+      but require more maintenance and setup. Each of these relies on the backing
+      filesystem being configured correctly.
+    - The `vfs` storage driver is intended for testing purposes, and in situations
+      where no copy-on-write filesystem can be used. Performance of this storage
+      driver is low, and because it provides no deduplication, will result in
```

Comment:
It still dedupe when copyfilerange is available?

Reply:
Oh, good one. Yeah, we can remove this bit (my main reason for mentioning it was to discourage "regular" users from using it; I know there's specific use-cases for using vfs, but for most users it should not something to look at)

Reply:
fixed